src/integrations/intercom/services/conversations-getter.js

```diff
@@ -51,56 +40,61 @@ function ConversationsGetter(Implementation, params, opts, collectionName) {
       });
   }
 
-  this.perform = () => {
-    model = Implementation.getModels()[collectionName];
-
-    return Implementation.Intercom.getCustomer(model, params.recordId)
-      .then((customer) => intercom.conversations
-        .list({
-          email: customer.email,
-          type: 'user',
-          display_as: 'plaintext',
-        })
-        .then((response) => {
-          const { conversations } = response.body;
-
-          if (response.body.pages.next) {
-            return fetchPages(response.body.pages, conversations);
-          }
-          return conversations;
-        })
-        .then((conversations) => [conversations.length,
-          conversations.slice(getSkip(), getSkip() + getLimit())])
-        .spread((count, conversations) => intercom.admins.list()
+  // NOTICE: Forcing the usage of bluebird to avoid issues with the spread on route file
+  this.perform = () => P.resolve(
+    ContactGetter
+      .getContact(intercom, Implementation, mappingValue, params.recordId)
+      .then((contact) => {
+        if (!contact.body.data) {
```

Comment:
`!contact || !contact.body || !contact.body.data` ?
As you did in the "detail service" ?
Without it, I think it could raise an error here if no mapping value is found.

Reply:
I think this is what generate this ugly log
<img width="691" alt="Screenshot 2020-04-10 at 18 21 56" src="https://user-images.githubusercontent.com/664261/79005940-400aa780-7b58-11ea-9717-7feef3c04d78.png">